tools/topology/sof/pipe-highpass-capture.m4

```diff
@@ -45,13 +45,13 @@ W_DATA(DEF_PGA_CONF, DEF_PGA_TOKENS)
 # IIR configuration
 #
 
-define(DEF_EQIIR_COEF, concat(`eqiir_coef_', PIPELINE_ID))
-define(DEF_EQIIR_PRIV, concat(`eqiir_priv_', PIPELINE_ID))
-
 # By default, use 40 Hz highpass response with 0 dB gain
 ifdef(`PIPELINE_FILTER1', , `define(PIPELINE_FILTER1, eq_iir_coef_highpass_40hz_0db_48khz.m4)')
+define(DEF_EQIIR_PRIV, PIPELINE_FILTER1)
 include(PIPELINE_FILTER1)
 
+define(DEF_EQIIR_COEF, concat(`eqiir_coef_', PIPELINE_ID))
+
 # EQ Bytes control with max value of 255
 define(DEF_EQIIR_COEF, concat(`eqiir_bytes_', PIPELINE_ID))
```

Comment:
This redefine if DEF_EQIIR_COEF is confusing (above define added). As result the .conf for IIRs 2.0 4.0 and 10.0 11.0 look different: coef vs. bytes.

The same coefficients are defined twice in .conf but I suppose it doesn't harm since the last "wins" I assume. The different coefficients sets will have at least different path/filename.

Reply:
@singalsu the macros are a bit messed up. I should remove the first define in a separate PR and only keep the second in another PR

Reply:
@ranj063 The double define seems to be present in this PR version too. I doubt there's need for two PRs to fix this. I could not spot any other other issues when looking the m4 and conf stuff.

(The "SectionControlBytes.eqiir_bytes" vs. "SectionControlBytes.eqiir_coef" confusion has been in in git main already for a while, not caused by this PR.)

Reply:
I meant this duplicated define in pipe-highpass-capture.m4;

```
define(DEF_EQIIR_COEF, concat(`eqiir_coef_', PIPELINE_ID))
# EQ Bytes control with max value of 255
define(DEF_EQIIR_COEF, concat(`eqiir_bytes_', PIPELINE_ID))
```